newIDE/app/src/KeyboardShortcuts/ShortcutsListRow.js

```diff
@@ -36,22 +35,21 @@ const ShortcutsListRow = (props: Props) => {
       />
       <ListItemSecondaryAction>
         {props.isClashing && (
-          <IconButton tooltip={t`This shortcut clashes with another action.`}>
+          <IconButton tooltip={props.i18n._(t`This shortcut clashes with another action.`)}>
```

Comment:
tooltip takes a MessageDescriptor (i.e: a `t` macro), not a string: https://github.com/4ian/GDevelop/blob/8a8adf213ada4749bcf38a7b1d5965096357fcf5/newIDE/app/src/UI/IconButton.js#L45